### PR TITLE
[ICDS Dashboard] add iodised salt indicator

### PR DESCRIPTION
**Pull request by Rohit25negi**
This add one field to the agg_awc table for the MPR report which comes from infra form

### CODE REVIEW

**Comment by Rohit25negi** [comment]
@calellowitz  humble ping.

File: custom/icds_reports/mpr_sqldata.py [hunk 1]
```diff
@@ -431,6 +433,34 @@ def rows(self):
...
+                filters['aggregation_level'] -= 1
+
+            filters['month'] = date(self.config['year'], self.config['month'], 1)
+
+            awc_data = AggAwc.objects.filter(**filters).values(
+                'use_salt',
+                'num_launched_awcs').order_by('month').first()
+
+            use_salt = awc_data.get('use_salt', 0)
+            percent = "%.2f" % ((use_salt or 0) * 100 / float(awc_data.get('num_launched_awcs', 0) or 1))
```
Comment (calellowitz) [inline custom/icds_reports/mpr_sqldata.py:455]: is `awc_number` in the previous definition only launched?

**Review by calellowitz — COMMENTED** [review]
One question but looks fine. Same idea as other about the new class maybe not being necessary

File: custom/icds_reports/mpr_sqldata.py (hunk 1 above)
Comment (Rohit25negi) [reply custom/icds_reports/mpr_sqldata.py:455]: hey, no they were the total number of awcs in that area(launched+unlaunched).
While going through the MPR calculations. It was decided to just include launched AWC as the denominator of this

**Comment by Rohit25negi** [comment]
> Same idea as other about the new class may be not being necessary

hey yeah, that was my first preference. But I think the structure of the MPR is such that I had to fall back to creating a new class. Reason explained here: https://github.com/dimagi/commcare-hq/pull/28093#issuecomment-674751840

**Comment by Rohit25negi** [comment]
Closing this as this is taken forward here https://github.com/dimagi/commcare-icds/pull/102

**Pull request closed without merging**